include/vm_phys.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _VM_PHYS_H_
+#define	_VM_PHYS_H_
+
+#include <queue.h>
+#include <common.h>
+
+#define VM_NFREEORDER 9
+#define VM_RESERVED 1
+
+typedef uintptr_t vm_addr_t;
+typedef uintptr_t vm_paddr_t;
+typedef struct vm_page* vm_page_t;
```

Comment:
Please do not. You're forcing a reader to remember that `vm_page_t` is not simply derived from `struct vm_page` but it's actually a pointer to `struct vm_page`. It's evil.